.github/workflows/ci.yml

```diff
@@ -46,11 +46,12 @@ jobs:
       - name: Check License Header
         uses: apache/skywalking-eyes/header@v0.6.0
 
-      - name: Install cargo-sort
-        run: make install-cargo-sort
-
       - name: Install taplo-cli
-        run: make install-taplo-cli
+        uses: taiki-e/install-action@v2
+        with:
+          tool: taplo-cli@0.9.3
```

Comment:
Bump version to fix this issue
<img width="810" alt="image" src="https://github.com/user-attachments/assets/a8e2a572-d299-4e05-8ce2-4c2bf835bbb0" />